src/main/java/org/apache/ibatis/reflection/Reflector.java

```diff
@@ -113,22 +114,25 @@ private void resolveGetterConflicts(Map<String, List<Method>> conflictingGetters
         } else if (winnerType.isAssignableFrom(candidateType)) {
           winner = candidate;
         } else {
-          throw new ReflectionException(
-              "Illegal overloaded getter method with ambiguous type for property "
-                  + propName + " in class " + winner.getDeclaringClass()
-                  + ". This breaks the JavaBeans specification and can cause unpredictable results.");
+          isAmbiguous = true;
+          break;
         }
       }
-      addGetMethod(propName, winner);
+      if (winner != null) {
```

Comment:
Is there case that the `winner` become `null` as actually?

Reply:
You're right. `winner` won't be `null`. 👍